iroh-qrpc-client/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "iroh-qrpc-client"
+version = "0.2.0"
+edition = "2021"
+authors = ["Friedel Ziegelmayer <me@dignifiedquire.com>", "ramfox"]
```

Comment:
we probably should have some kind of `team@` or similar?  At sentry we used `oss@`.

Reply:
Our team is small enough that we might as well add the individual names...